components/dropdown/demo/overlay-visible.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 6
-title: 菜单隐藏方式
+title:
+  zh-CN: 菜单隐藏方式
+  en-US: The way of hiding menu.
 ---
 
+## zh-CN
+
 默认是点击关闭菜单，可以关闭此功能。
 
+## en-US
+
+The default is to close the menu when you click on menu items, this feature can be turned off.
```

Comment:
I think the `default` can be treated as a noun this case besides adj. ~
Also there are lots of examples:

<img width="726" alt="2016-06-29 4 41 32" src="https://cloud.githubusercontent.com/assets/7017767/16446465/519f377e-3e1a-11e6-855e-7644c3b3512e.png">